packages/emotion/package.json

```diff
@@ -25,7 +25,7 @@
   "license": "MIT",
   "dependencies": {
     "@babel/runtime": "^7.13.10",
-    "@emotion/core": "^10.0.35",
+    "@emotion/react": "11.0.0",
```

Comment:
I think we should use the `^` syntax so we are getting the features and bugfixes in version 11.

Reply:
Since we plan to do constant version bumps, we should know explicitly the version we are on to avoid unnecessary complications

Reply:
I don't see how automatically getting the minor and patch versions for a given package would complicate anything with our constant version bumps, on the contrary, I think it would make our lives easier since then we would not need to have to update the minor and patch versions manually every time we do this reoccuring update task.